lib/actuators/include/Actuator.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+/*
+ * The Actuator class is an abstracted class that encompasses
+ * DC motors, brushless motors, limit servos, and continuous servos.
+ * What "value" represents depends on the actuator type:
+ *
+ *      Actuator Type       Value
+ *      ------------------------------
+ *      DC Motor            Speed
+ *      Brushless Motor     ???
+ *      Limit Servo         Position
+ *      Continuous Servo    Speed
+ */
+
+class Actuator {
```

Comment:
purely virtual interfaces really need a virtual destructor: https://stackoverflow.com/questions/3628529/should-c-interfaces-have-a-virtual-destructor